PropertySheets/Chakra.Cpp.props

```diff
@@ -35,7 +35,7 @@
       <CallingConvention Condition="'$(CallingConvention)' != ''">$(CallingConvention)</CallingConvention>
     </ClCompile>
     <Link>
-      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
```

Comment:
What's the difference in the generated debug info? I just want to know why you're making the change.

Reply:
Dependent projects require using only full PDBs and if only partial ones are encountered, they get converted in-place which, from what we've observed, essentially hangs the build/test process for those projects.

The alternative to this is just to not include the PDB, which I think is not the best course of action.